src/screens/LoginPage/LoginPage.spec.tsx

```
@@ -312,6 +363,231 @@ describe('Testing Login Page Screen', () => {
+      () => {
+        expect(screen.getByPlaceholderText(/Name/i)).not.toBeDisabled();
+        expect(screen.getByTestId(/signInEmail/i)).not.toBeDisabled();
+        expect(screen.getByPlaceholderText('Password')).not.toBeDisabled();
+        expect(
+          screen.getByPlaceholderText('Confirm Password'),
+        ).not.toBeDisabled();
+        expect(screen.getByTestId('registrationBtn')).not.toBeDisabled();
+      },
+      { timeout: 1000 },
+    );
+  });
```

ESLint

[error] Test is used multiple times in the same describe(suite) block

(vitest/no-identical-title)